django/conf/project_template/project_name/urls.py

```diff
@@ -2,9 +2,15 @@
 from django.contrib import admin
 
 urlpatterns = [
-    # Examples:
-    # url(r'^$', '{{ project_name }}.views.home', name='home'),
-    # url(r'^blog/', include('blog.urls')),
+    # Example 1: Function Views
```

Comment:
I would like to have the word 'Example' or examples somewhere, as I think that this makes it as clear as possible for beginners.